content.js

```diff
@@ -73,7 +73,7 @@ function addToggleBtnEvents() {
 }
 
 function label() {
-	return toggleOn ? 'Show dotfiles' : 'Hide dotfiles';
+	return toggleOn ? 'Show dot' : 'Hide dot';
```

Comment:
Instead of this, can you just make the URL box more narrow: https://github.com/sindresorhus/hide-files-on-github/blob/29fcb06efad86da18b9d0077c2e8c3c5d55b716b/css/content.css#L5 ?